src/main/java/ch/jalu/configme/resource/YamlFileResource.java

```diff
@@ -70,6 +72,16 @@ public void exportProperties(@NotNull ConfigurationData configurationData) {
             rootNode = root.convertToNode(nodeBuilder);
         }
 
+        List<String> footerStrings = configurationData.getAllComments().get(CommentsConfiguration.FOOTER_KEY);
```

Comment:
`configurationData.getCommentsForSection(CommentsConfiguration.FOOTER_KEY)` should be easier to access (returns empty list and never null), might make your life easier. Could even then drop the `if` altogether below this comment :) But no strong emotions